src/components/TransactionsInterpreter/helpers/getDenominatedValue.ts

```diff
@@ -0,0 +1,30 @@
+import { UITransactionType } from './types';
+import { networkConfigSelector } from 'reduxStore/selectors';
+import { store } from 'reduxStore/store';
+import { denominate } from 'utils';
+import { DenominationConfig } from '../TransactionsInterpreter';
+
+export function getDenominatedValue(
+  transaction: UITransactionType,
+  denominationConfig: DenominationConfig
+) {
+  const networkConfig = networkConfigSelector(store.getState());
```

Comment:
do not use store.getState because function will not be testable. Or create  `useGetDenominatedValue`(